examples/values_use_a_custom_default_scheduling_topology.yaml

```diff
@@ -10,10 +10,7 @@ command:
   if ! grep -q profiles $CONFIG; then
     ## ADDING PodTopologySpread PROFILE IN SCHEDULER CONFIGURATION
     echo "Adding scheduler plugin config to 'PodTopologySpread'"
-    printf "profiles:\n  - schedulerName: default-scheduler\n    pluginConfig:\n    - name: PodTopologySpread\n    args:\n      defaultConstraints:\n        - maxSkew: 1\n          topologyKey: topology.kubernetes.io/hostname\n          whenUnsatisfiable: ScheduleAnyway\n      defaultingType: List" >> $CONFIG
-
-    ## DISABLE SELECTOR SPREAD PLUGIN IN SCHEDULER CONFIGURATION
-    printf "\n    plugins:\n      score:\n        disabled:\n        -name: SelectorSpread\n" >> $CONFIG
+    printf "profiles:\n  - pluginConfig:\n    - name: PodTopologySpread\n      args:\n        defaultConstraints:\n          - maxSkew: 1\n            topologyKey: topology.kubernetes.io/hostname\n            whenUnsatisfiable: ScheduleAnyway\n        defaultingType: List" >> $CONFIG
```

Comment:
I'd split it into multiple `printf` statements like:

```
printf "profiles:\n"          >> $CONFIG
printf "  - pluginConfig:\n"  >> $CONFIG
...
```

Reply:
nice idea, done